styles/styles.css

```diff
@@ -22,29 +50,52 @@ table {
     margin:0 auto 80px;
 }
 
-.advantages__list {
-    list-style: none;
-    margin: 0 104px auto 104px;
+.advantages__col {
+    text-align: center;
+
 }
 
-.advantages__item {
-    display: inline-block;
-    width: 296px;
-    margin-right: 12px;
+.advantages__image {
+    margin: 0 auto;
 }
 
-.advantages__item:last-child{
-    margin-right: 0;
+.advantages__subtitle {
+    margin-top: 48px;
+    margin-bottom: 122px;
```

Comment:
It's your choice, but I prefer to add some margin-bottom to the wrapper to prevent sticking of any content to the next block.

Reply:
What value would be acceptable to add?

Reply:
It is 120px in the design and it is good to add it actually to the wrapper. Otherwise, you have to think about its content every time it changes.